src/components/Register/Specials.js

```diff
@@ -1,15 +1,47 @@
-import React from 'react';
-import { VStack, Checkbox, Text } from 'native-base';
+import React, { useState } from 'react';
+import { VStack, Checkbox, Text, Modal } from 'native-base';
+import { handleFailedPolicies } from '../utilities/failedPolicies';
 
-const Specials = ({ label, val, setter, terms = null }) => {
+function TermsModal({ terms, showModal, setModal }) {
   return (
-    <VStack>
+    <React.Fragment>
+      <Text
```

Comment:
I'm thinking to make theming easier, we should create components that represent the text type, like h1, h2, h3, paragraph ...